libasi/99-asi.rules

```diff
@@ -1,3 +1,3 @@
-ACTION=="add", ATTR{idVendor}=="03c3", RUN+="/bin/sh -c '/bin/echo 256 >/sys/module/usbcore/parameters/usbfs_memory_mb'"
-# All ASI Cameras and filter wheels
-SUBSYSTEMS=="usb", ATTR{idVendor}=="03c3", MODE="0666" 
+ACTION=="add", ATTR{idVendor}=="03c3", RUN+="/bin/sh -c '/bin/echo 200 >/sys/module/usbcore/parameters/usbfs_memory_mb'"

```

Comment:
Any reason for this?

Reply:
That's what was in the asi.rules file in the SDK. I did this because the allocated USB memory is less and I figured that ZWO have a reason for it. I can revert it if you want.

Reply:
Right, it should remain 256 to be consistent at least with the rest of INDI rule files.

Reply:
OK, done.